wagtail/admin/templates/wagtailadmin/chooser/_browse_results.html

```diff
@@ -1,20 +1,16 @@
 {% load i18n wagtailadmin_tags %}
 
 {% if show_locale_labels %}
-    <div class="locale-options">
+    <div>
         {% if locale_options %}
             {% if selected_locale %}
                 {% include "wagtailadmin/shared/locale_selector.html" with translations=locale_options locale=selected_locale only %}
             {% else %}
                 {% include "wagtailadmin/shared/locale_selector.html" with translations=locale_options locale=parent_page.locale only %}
             {% endif %}
         {% else %}
-            <div class="c-dropdown t-inverted {{ class }}" style="display: inline-block;">
-                <div class="c-dropdown__button u-btn-current">
-                    {% icon name="site" classname="default" %}
-                    {{ parent_page.locale.get_display_name }}
-                </div>
-            </div>
+            {% icon name="site" classname="default" %}
+            {{ parent_page.locale.get_display_name }}
```

Comment:
This was rendering something that looked like a dropdown but wasn’t actually interactive. So I chose to use the same "text-only" style as inside `locale_selector.html`.